beast/observationmodel/phot.py

```diff
@@ -733,3 +733,44 @@ def appendVegaFilter(filtInst, VegaLib=__default_vega__):
     sedTab.flush()
     vtab.close()
     print("% {0}: Filter {1} added to {2}".format(sys.argv[0], filtInst.name, VegaLib))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "filter_throughput_curve",
+        type=str,
+        default=None,
+        help="filename of filter throughput curve to be added into the BEAST filter library",
+    )
+    parser.add_argument(
+        "--tablename",
+        type=str,
+        help="Table name for the filter",
+    )
+    parser.add_argument(
+        "--observatory",
+        type=str,
+        help="Observatory for the filter (e.g., HST, GALEX)",
+    )
+    parser.add_argument(
+        "--instrument",
+        type=str,
+        help="Instrument associated with the filter (WFC3, ACS, ...)",
+    )
+    parser.add_argument(
+        "--filtername",
+        type=str,
+        help="Name for the filter",
+    )
+
+    args = parser.parse_args()
+
+    if args.filter_throughput_curve:
+        lamb, throughput = numpy.loadtxt(args.filter_throughput_curve, usecols=(0, 1),
+                                         unpack=True)
+        append_filter(lamb, throughput, args.tablename, args.observatory, args.instrument,
+                      args.filtername)
+
+    if not any(vars(args).values()):
+        parser.print_help()
```

Comment:
I think this should be a separate script in the tools subdir instead of added to phot.py.   For consistency with how similar capabilities are handled in the BEAST.

Reply:
That makes sense. Will start a separate script for this functionality.

Reply:
This task could be merged with the outstanding WIP PR #646!

Reply:
Need to add this script to this PR.